cpp/include/cuml/neighbors/knn.hpp

```diff
@@ -97,17 +97,18 @@ class kNN {
   int total_n;
   int indices;
   int D;
-  bool verbose;
```

Comment:
removed this because the verbose flag is directly used to set the logging level in the ctor.